Runtime/Scripts/MobiledgeXIntegration.cs

```diff
@@ -125,6 +125,25 @@ public MobiledgeXIntegration(CarrierInfo carrierInfo = null, NetInterface netInt
             melMessaging = new MelMessaging(appName);
             matchingEngine.SetMelMessaging(melMessaging);
         }
+        
+        /// <summary>
+        /// Constructor for MobiledgeXIntegration. This class has functions that wrap DistributedMatchEngine functions for ease of use
+        /// </summary>
+        public MobiledgeXIntegration(string OrgName , string AppName , string AppVers , string DeveloperAuthToken = "")
```

Comment:
Also, I think we have been using lowercase first letter for parameters (ie. string orgName, string appName, etc.). When setting these parameters, because they are the same as your class variables, you can just do: 

```
this.orgName = orgName;
this.appVers = appVers;
this.appName = appName;
```